README.md

```diff
@@ -1 +1,43 @@
-# Marketing-Melons-grp-project-
\ No newline at end of file
+# Marketing-Melons-grp-project-
+
+Market place is where we have information about melons ,bidding process involved.
+
+Clone the project
+
+git clone https://github.com/deeptidhar/Marketing-Melons-grp-project-.git
+Go to the project directory
+
+cd forever-project
+virtualenv env
+source env/bin/activate
+
+Install requirements
+ pip3 install requirements.txt
```

Comment:
`pip3 install -r requirements.txt`

Reply:
In the future, you can also add running the seed file (which creates the database, tables, and example data) and then running the server.